pkg/cmd/cli/cmd/create/route.go

```diff
@@ -409,3 +430,10 @@ func resolveRouteName(args []string) (string, error) {
 	}
 	return "", nil
 }
+
+func resolveWildcardPolicy(wildcardpolicy string) (string, error) {
+	if len(wildcardpolicy) > 0 && (wildcardpolicy != "Subdomain" && wildcardpolicy != "None") {
+		return "", fmt.Errorf("only \"Subdomain\" or \"None\" are supported for wildcardpolicy")
+	}
+	return wildcardpolicy, nil
+}
```

Comment:
The validation functions are called as part of object creation on the server side. I used debug functions to print a stack trace on the server side on a failed attempt to create a route (for another PR but should be the same for this one) [StackTrace](http://pastebin.com/YcRLdKyA)